### PR TITLE
Added Integrated MakeTrip Project

### PR DESCRIPTION
**Pull request by ParasY1724**

### CODE REVIEW

**Comment by piyush960** [comment]
Code Submission of team CodeSnap for Credenz'24 Web Weaver at PICT

**Review by gautamgambhir97 — COMMENTED** [review]
Contributing
https://github.com/fetchai/uAgents/blob/main/integrations/CONTRIBUTING.md

Include a project.json file, example of:

```
{
    "title": "Bert Base Uncased",
    "description": "BERT base model (uncased) Pretrained model on English language using a masked language modeling (MLM) objective.",
    "categories":    ["Text Classification", "Hugging Face", "Text Generation"], 
    "deltav": false
}
```
Set deltav to true if your agent is built using agentverse and is accessible by DeltaV.

File: integrations/MakeTrip_PICT_Codesnap/agents/flight_agent.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        "destinationEntityId": dest_entityId,
+        "date": date,
+        "adults": person,
+        "currency": "USD",
+        "market": "en-US",
+        "countryCode": "US"
+    }
+
+    headers = {
+        "X-RapidAPI-Key": "1dc3581a84mshdcf979fd41a22a8p1bcc2ajsnd841021890c1",
```
Comment (gautamgambhir97) [inline integrations/MakeTrip_PICT_Codesnap/agents/flight_agent.py:92]: API Key exposed.

Comment (piyush960) [reply integrations/MakeTrip_PICT_Codesnap/agents/flight_agent.py]: now fixed

**Review by cmaliwal — APPROVED** [review]

**Review by piyush960 — COMMENTED** [review]
now fixed

**Review by gautamgambhir97 — APPROVED** [review]

**Pull request merged**